backend/src/graphql/resolver/util/validateAlias.ts

```diff
@@ -3,6 +3,9 @@ import { User as DbUser } from '@entity/User'
 
 import { LogError } from '@/server/LogError'
 
+// eslint-disable-next-line security/detect-unsafe-regex
+export const validAliasRegex = /^(?=.{3,20}$)[a-zA-Z0-9]+(?:[_-][a-zA-Z0-9]+?)*$/
```

Comment:
i would write this in caps since its a constant, but thats a minor non blocking thing
```suggestion
export const VALID_ALIAS_REGEX = /^(?=.{3,20}$)[a-zA-Z0-9]+(?:[_-][a-zA-Z0-9]+?)*$/
```